From 5faa35d0b8cc1403acb6760e9e95905cfd691872 Mon Sep 17 00:00:00 2001
Message-Id: <5faa35d0b8cc1403acb6760e9e95905cfd691872.1651807732.git.geliang.tang@suse.com>
From: Muhammad Usama Anjum <usama.anjum@collabora.com>
Date: Wed, 19 Jan 2022 15:15:29 +0500
Subject: [PATCH] selftests: mptcp: Add the uapi headers include variable
Git-commit: 5faa35d0b8cc1403acb6760e9e95905cfd691872
Patch-mainline: v5.18-rc1
References: bsn#2
Modified-by-SEL: No


Out of tree build of this test fails if relative path of the output
directory is specified. Add the KHDR_INCLUDES to correctly reach the
headers.

Signed-off-by: Muhammad Usama Anjum <usama.anjum@collabora.com>
Reviewed-by: Matthieu Baerts <matthieu.baerts@tessares.net>
Signed-off-by: Shuah Khan <skhan@linuxfoundation.org>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 tools/testing/selftests/net/mptcp/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/testing/selftests/net/mptcp/Makefile b/tools/testing/selftests/net/mptcp/Makefile
index 0356c4501c99..f905d5358e68 100644
--- a/tools/testing/selftests/net/mptcp/Makefile
+++ b/tools/testing/selftests/net/mptcp/Makefile
@@ -3,7 +3,7 @@
 top_srcdir = ../../../../..
 KSFT_KHDR_INSTALL := 1
 
-CFLAGS =  -Wall -Wl,--no-as-needed -O2 -g  -I$(top_srcdir)/usr/include
+CFLAGS =  -Wall -Wl,--no-as-needed -O2 -g -I$(top_srcdir)/usr/include $(KHDR_INCLUDES)
 
 TEST_PROGS := mptcp_connect.sh pm_netlink.sh mptcp_join.sh diag.sh \
 	      simult_flows.sh mptcp_sockopt.sh
-- 
2.34.1

